app/models/drawing.rb

```diff
@@ -2,6 +2,9 @@ class Drawing < ActiveRecord::Base
   enum status: %i(pending complete)
   enum gender: %i(not_specified female male other)
 
+  STAGES_OF_JOURNEY = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"].freeze
```

Comment:
It's generally for readability and old rules stemming from C++ why constants tend to be at the very top of the class. 

For readability and conveying to convention, it's worth putting the constants above the enums.